hogql_parser/parser.cpp

```diff
@@ -452,6 +462,110 @@ class HogQLParseTreeConverter : public HogQLParserBaseVisitor {
     return ret;
   }
 
+  VISIT(ThrowStmt) {
+    PyObject* expr;
+    try {
+      expr = visitAsPyObjectOrNone(ctx->expression());
+    } catch (...) {
+      throw;
+    }
+    PyObject* ret = build_ast_node("ThrowStatement", "{s:N}", "expr", expr);
+    if (!ret) {
+      Py_DECREF(expr);
+      throw PyInternalError();
+    }
+    return ret;
```

Comment:
You should just do:
`RETURN_NEW_AST_NODE("ThrowStatement", "{s:N}", "expr", expr);`
– the decref is not needed, because:
https://github.com/PostHog/posthog/blob/9a5ab8fc0e25b29ed47c0d9bb0c3c7d99e5d9a40/hogql_parser/parser.cpp#L49-L52
In fact I think `Py_DECREF(expr)` here might cause a crash due to the refcount going negative